src/SingleCellAnnotator.imjoy.html

```diff
@@ -1457,8 +1495,12 @@
               </div>  
               <div v-for="(v, task_name) in s" :key="task_name">
                 <div class="card-subtitle text-gray">{{task_name}}</div>
+                <div :id="k + task_name">
+                  <span><button title="show chart" @click="barChartShow(v, k + task_name)">chart</button></span>
+                </div>
```

Comment:
Note you need to click to show the charts. pls suggest a way to auto run the func and show charts in this part